### ISSUE TITLE
False positive when named arguments in function call enclosed in quotes

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#210]
CFLint reports a `MISSING_SEMI`, exactly `End of statement(;) expected instead of )` for the following line of code:

```
foo.bar("argNameOne" = argValueOne, "argNameTwo" = argValueTwo);
```

Seems to be triggered by the argument names enclosed in quotes, which to the best of my knowledge is valid ColdFusion code.

**Comment by ryaneberly** [comment]
fixed in cfparser.
